core/jobs_test.py

```diff
@@ -864,7 +874,7 @@ def test_raise_error_with_get_batch_job_manager_class(self):
                 'Subclasses of BaseContinuousComputationManager must implement '
                 '_get_batch_job_manager_class(). This method should return the'
                 'manager class for the continuously-running batch job.')):
-            jobs.BaseContinuousComputationManager._get_batch_job_manager_class()  # pylint: disable=protected-access
+            jobs.BaseContinuousComputationManager.stop_computation('admin_user_id')   #_get_batch_job_manager_class()  # pylint: disable=protected-access
```

Comment:
Why is `_get_batch_job_manager_class` commented out? Remove that piece of code if it isn't used.